Configuration/DataProcessing/python/Impl/ppEra_Run3_2024_UPC.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+"""
+_ppEra_Run3_2024_UPC_
+Scenario supporting UPC collisions for 2024
+"""
+
+import os
+import sys
+
+from Configuration.DataProcessing.Reco import Reco
+import FWCore.ParameterSet.Config as cms
+from Configuration.Eras.Era_Run3_2024_UPC_cff import Run3_2024_UPC
+
+from Configuration.DataProcessing.Impl.pp import pp
+
+class ppEra_Run3_2024_UPC(pp):
+    def __init__(self):
+        pp.__init__(self)
+        self.recoSeq=''
+        self.cbSc='pp'
```

Comment:
@mandrenguyen, IIUC the datasets that now you would be processing with this scenario are the ones that last year were processed with `hiForwardScenario = "ppEra_Run3_2023_repacked"`.
That scenario has 

https://github.com/cms-sw/cmssw/blob/2647f990f5bf968ff41bdc59844e73e3fc34ecd9/Configuration/DataProcessing/python/Impl/ppEra_Run3_2023_repacked.py#L23

why is it not implemented here?

Reply:
ah, yes, thanks for spotting it. 
I guess that's in relation to this failure right? :
 https://github.com/cms-sw/cmssw/issues/46186#issuecomment-2387598924
Will fix.

Reply:
fix is here:  https://github.com/cms-sw/cmssw/pull/46194